boards/we/orthosie1ev/we_orthosie1ev.yaml

```diff
@@ -0,0 +1,23 @@
+identifier: we_orthosie1ev
+name: we_orthosie1ev
+type: mcu
+arch: riscv
+toolchain:
+  - zephyr
+supported:
+  - adc
+  - gpio
+  - i2c
+  - i2s
+  - watchdog
+  - uart
+  - dma
+  - pwm
+  - spi
+  - counter
+  - entropy
+testing:
+  ignore_tags:
+    - net
```

Comment:
Will need to be removed if/when https://github.com/zephyrproject-rtos/zephyr/pull/80785 gets merged